src/object/srv_obj_migrate.c

```diff
@@ -552,8 +552,7 @@ migrate_fetch_update_inline(struct migrate_one *mrone, daos_handle_t oh,
 	if (fetch) {
 		rc = dsc_obj_fetch(oh, mrone->mo_epoch, &mrone->mo_dkey,
 				   mrone->mo_iod_num, mrone->mo_iods, sgls,
-				   NULL, DIOF_TO_LEADER | DIOF_FOR_MIGRATION,
-				   NULL);
+				   NULL, DIOF_FOR_MIGRATION, NULL);
```

Comment:
We need to distinguish rebuild case from migration. For rebuild case, the DTX resync on different servers run in parallel, when the rebuild logic comes here on current server, it does NOT means that the DTX resync on other servers have done. So here, we still need to fetch from leader for rebuild case.

Reply:
Ah, right. Thanks.